devtools/azure/linux-dependencies.yml

```
@@ -1,18 +1,26 @@
 parameters:
-  rustup_toolchain: ''
-  node_version: ''
+  rustup_toolchain: ""
+  node_version: ""
 steps:
-- script: |
-    sudo apt install -y build-essential
-- script: |
-    curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain ${{ parameters.rustup_toolchain }}
-    set PATH=$PATH:~/.cargo/bin
-  displayName: Install rust
-- script: |
-    rustc --version
-    cargo --version
-  displayName: Test/query binaries
-- task: NodeTool@0
-  inputs:
-    versionSpec: ${{ parameters.node_version }}
-  displayName: 'Install Node.js'
+  - script: |
+      sudo apt install -y build-essential
+  - script: |
+      curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain ${{ parameters.rustup_toolchain }}
+      set PATH=$PATH:~/.cargo/bin
+    displayName: Install rust
+  - script: |
+      rustc --version
+      cargo --version
+    displayName: Test/query binaries
+  - script: |
+      curl -O -L "https://github.com/nervosnetwork/ckb-indexer/releases/download/v0.2.2/ckb-indexer-0.2.2-linux.zip"
```

```suggestion
      curl -O -L "https://github.com/nervosnetwork/ckb-indexer/releases/download/v{{ parameters.ckb_indexer_version}}/ckb-indexer-{{ parameters.ckb_indexer_version}}-linux.zip"
```

use the [azure pipeline parameters](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/runtime-parameters?view=azure-devops&tabs=script#use-parameters-in-pipelines) instead of

---

done